action.yml

```diff
@@ -43,6 +43,9 @@ inputs:
   mappingFile:
     description: "The mapping.txt file used to de-obfuscate your stack traces from crash reports"
     required: false
+  changesNotSentForReview:
+    description: "If the build should be sent directly to release"
+    required: true
```

Comment:
Adding a new "required" parameter will break all existing usages of this action.
This is probably one of the reasons this hasn't been accepted.
This should be changed to required:false, both here and below in main.ts, and the default value should enable the action to work as it has until now.